.github/workflows/commit-validation.yml

```
@@ -56,13 +97,14 @@ jobs:
       node-version: ${{matrix.node-version}}
 
   tests-windows:
-    needs: build
+    if: contains(needs.detect-changes.outputs.changes, 'src-or-tests')
+    needs: [detect-changes, build]
     uses: ./.github/workflows/tests-windows.yml
     with:
       node-version: 20
 
-  # Run with most databases possible to provide the coverage of the tests
   coverage:
```

actionlint (1.7.8)

property "detect-changes" is not defined in object type {tests-linux: {outputs: {}; result: string}; tests-windows: {outputs: {}; result: string}}

(expression)